.buildkite/pipeline-scheduler.py

```diff
@@ -73,7 +73,8 @@ def get_release_branches() -> typing.List[str]:
     try:
         release_branches = [branch for branch in resp["branches"]]
     except KeyError:
-        fail_with_error(f'''Didn't find the excepted structure ["branches"] in the response [{resp}] from [{ACTIVE_BRANCHES_URL}]''')
+        fail_with_error(
+            f'''Didn't find the excepted structure ["branches"] in the response [{resp}] from [{ACTIVE_BRANCHES_URL}]''')
```

Comment:
nit: this is out of scope for this PR do you think we could undo it? Changing this, due to the CODEOWNERS file, resulted in requesting a review from an additional team (ingest-eng-prod) which isn't otherwise required for this PR.